Makefile

```diff
@@ -142,6 +142,13 @@ $(GOPATH)/bin/golangci-lint:
 lint: $(GOPATH)/bin/golangci-lint
 	$(GOPATH)/bin/golangci-lint run
 
+$(GOPATH)/bin/gosec:
+	GO111MODULE=on go get github.com/securego/gosec/cmd/gosec@2.0.0
+
+# Run gosec against code
+gosec: $(GOPATH)/bin/gosec
+	$(GOPATH)/bin/gosec -tests -severity medium  ./...
+
```

Comment:
I'd really prefer we invoke this through golangci-lint rather than adding our own target for it.

Reply:
@cfergeau I had another look to the way `golangci-lint` try to use the `gosec` and I think we should use the `gosec` from `securego/gosec` side instead from a fork which was updated last on 2nd Feb.

gosec linter which is part of golangci-lint coming from `github.com/golangci/gosec` instead `github.com/securego/gosec` https://github.com/golangci/golangci-lint/blob/master/pkg/golinters/gosec.go#L11 , which is last updated on 2nd Feb https://github.com/golangci/gosec/tree/02.2019  ( https://github.com/golangci/golangci-lint/blob/master/go.mod#L23) which means this is out dated.

Reply:
They just updated it, dunno if that's related to something you did ?
https://github.com/golangci/golangci-lint/pull/681/commits/150843707f6fd57328e089abf5ae520541503528